src/main/java/frc/robot/Robot.java

```diff
@@ -24,20 +25,21 @@
  // project.
 
 public class Robot extends TimedRobot {
-  private Command m_autonomousCommand;
-  private RobotContainer m_robotContainer;
-  private TabContainer m_tabContainer;
-
-  // This function is run when the robot is first started up and should be used for any
-  // initialization code.
+  private Command autonomousCommand;
+  private RobotContainer robotContainer;
+  private TabContainer tabContainer;
+  private IntakeSubsystem ballController;
```

Comment:
Intake subsystem should be exist and be created inside of robot container, not inside Robot.java.